enterprise/backend/src/metabase_enterprise/serialization/api.clj

```diff
@@ -86,10 +86,7 @@
                        report)
                      (catch Exception e
                        (reset! err e)
-                       (log/errorf "Error during serialization: %s %s"
-                                   (ex-message e)
-                                   (-> (ex-data e)
-                                       (dissoc :toucan2/context-trace))))))]
+                       (serdes/log-stripped-error "Error during serialization" e))))]
```

Comment:
nit: "Error during serialization" is repeated a few times, maybe it should be a default arg.

Reply:
sometimes it's "Error during deserialization" :)